ui/src/pages/Results.js

```diff
@@ -175,18 +185,49 @@ class SearchResultImpl extends Component {
   }
 
   render() {
+    var nameStr = this.props.result.fullNames.join(', ');
+    if (this.props.result.alternateNames.length > 0) {
+      nameStr += ' (' + this.props.result.alternateNames.join(', ') + ')';
+    }
+    const formattedTimestamp = <FormattedDate
+        value={new Date(this.props.result.timestamp + 'Z')}
```

Comment:
Maybe guarantee a valid ISO 8601 timestamp from the API so you don't have to know about appending 'Z'?

Reply:
Done.